README.md

```diff
@@ -1,157 +1,160 @@
-By changing the collateral in the `select` element, the user must see the amount and installment options corresponding to the asset chosen. That is, when selecting either _"Home"_ or _"Auto"_, you must show the correspondent values for each option on the form fields and on the slider.
-Please find the corresponding values below:
+  response
 
-**Auto**
-- Minimum loan amount: R$ 3.000,00;
-- Maximum loan amount: R$ 100.000,00;
-- Number of installments: 24 / 36 / 48 months
-- Collateral minimum value: R$ 5.000,00;
-- Collateral maximum value: R$ 3.000.000,00;
+  ```javascript
+  {
+    seatNumber: number;
+  }
+  ```
 
-**Home**
-- Minimum loan amount: R$ 30.000,00;
-- Maximum loan amount: R$ 4.500.000,00;
-- Number of installments: 120 / 180 / 240 months
-- Collateral minimum value: R$ 50.000,00;
-- Collateral maximum value: R$ 100.000.000,00;
+- POST https://frontend-challenge-vlc-api.public.dev.creditas.io/api/snacks/confirm
 
-Finally, you must update the monthly payment value whenever any input changes.
+  request
 
-### Layout
-The template requires some minor css changes in order to be displayed as proposed in the references provided. Feel free to use any property you feel like to achieve so.
+  ```javascript
+  {
+    id: string;
+  }
+  ```
 
-Desktop version:
-![Desktop version](./desktop-layout.png)
+  response
 
-Mobile version:
-![Mobile version](./mobile-layout.png)
+  ```javascript
+  {
+    queueNumber: number;
+  }
+  ```
 
-### Display the help text
+### Layout
 
-To display the help text, we provide two endpoints on localhost:4000 ( for more details you can check server.js file ):
-- 'api/question' which responses is 
-```
-{ text: 'Question: Can you help me?' }
-```
-- 'api/answer' which response is 
-```
-{ text: 'Answer: Yes sure.' }
-```
+The template requires some css rules in order to be displayed as proposed in the screenshots provided. Feel free to use any property you feel like to achieve so.
 
-When the user clicks the help link, we need to display an alert with the following text:
+#### Desktop version
 
-*Question: Can you help me?.</br>Answer: Yes, sure*
+![Desktop version](./desktop-layout.png)
+
+#### Mobile version
 
-You will have to consider the following:
-- Each endpoint takes 10 seconds to respond, so it will be desirable to avoid the user to wait for this 10 seconds every time the help link is clicked. 
-- The requests can return a 500 http status, in this case you will have to display the message error provided in the response
+![Mobile version](./mobile-layout.png)
 
 ## Development
 
 ### Prerequisites
 
 You will need to install on your machine:
-- [NodeJs](https://nodejs.org/en/) in order to run the application.
+
 - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in order to clone the repository containing this challenge and submit the pull request with your solution.
 
 ### Configuration to run the environment
```

Comment:
Combined with @nico-creditas comment above, I think that we could remove this section as well